api/scpca_portal/models/dataset.py

```diff
@@ -678,6 +683,23 @@ def merged_projects(self) -> Iterable[Project]:
 
         return Project.objects.none()
 
+    @property
+    def multiplexed_projects(self) -> Iterable[Project]:
+        """
+        Returns all project instances which have multiplexed data.
+        """
+        # Multiplexed samples are not available with anndata
+        if self.format == DatasetFormats.ANN_DATA:
+            return Project.objects.none()
+
+        if self.is_ccdl:
+            return self.projects.filter(has_multiplexed_data=True)
```

Comment:
This can be removed

Reply:
@dvenprasad can you confirm that when possible portal wide downloads will always contain multiplexed samples. (We are not creating a portal-wide exclude multiplexed dataset).